tests/test_main.py

```diff
@@ -5,8 +5,10 @@
 
 import pytest
 
+import devtools.highlight
```

Comment:
looks like you have this imported twice with different names.

Reply:
The marked import is for the monkeypatch and the other for direct usage. If you want I can remove the other and update the usage.